pages/settings/wallets/nwc.js

```diff
@@ -47,6 +47,8 @@ export default function NWC () {
           initialValue={nwcUrl}
           label='connection'
           name='nwcUrl'
+          type='password'
+          autoComplete='new-password'
```

Comment:
[interesting, didn't know about `new-password`!](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#preventing_autofilling_with_autocompletenew-password)

makes sense to use it here